src/main/java/com/faforever/client/remote/AbstractServerAccessor.java

```diff
@@ -57,4 +57,6 @@ void close() throws IOException {
     stopped = true;
     IOUtils.closeQuietly(dataInput);
   }
+
+  protected abstract void onUIDNotFound(Exception e);
```

Comment:
This method is only called from the implementation itself and should therefore not be declared here, and private.